components/__snapshots__/size_aware_image.test.jsx.snap

```diff
@@ -38,7 +38,7 @@ exports[`components/SizeAwareImage should render a placeholder and has loader wh
   </div>
   <button
     aria-label="file thumbnail"
-    className="style--none"
+    className="style--none file-preview__button"
```

Comment:
@asaadmahmood I am not sure of the className. Let me know if you have any suggestions. Nother thing i think it is better going forward with css around `size_aware_image` is to have it based on classname rather than `tags` or `nesting of tags` as changing this is causing some scroll pops